### DISCUSSION TITLE
Format on save breaks angular html template

### DISCUSSION THREAD

**Discussion by ionescu77** [zed-industries/zed#18547] [Feature Requests]
Hello,

## Background

it is the second time I need to disable format on save.

I have already did this  previously CMD + SHIFT + P (command palette) and in zed `local project settings`

```
{
  "format_on_save": "off"
}

```

Probably after installing zed and refreshing after update it got lost.

---

## Issue

I am adding some features in some old Angular 9 project. I use Zed. Only and only with html templates in angular components, the format on save of the html will bring down the angular local server to a crash.

It is a relativelly big html file which contains angular logic. And the terminal output of the ng server is huge.

I do not have time to investigate as I need to deploy a fix.

I'll be back with an Issue when I identify why

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
